src/main/java/tri/stringreverse/StringReverse.java

```diff
@@ -0,0 +1,16 @@
+package tri.stringreverse;
+
+/**
+ * Output a reversed string.
+ */
+public class StringReverse {                //create class
+    public String reverse(String text){     //function named reverse, the function used a built-in reverse function and return the reversed string
+        return reverse(text);
```

Comment:
To reverse a string, write: return new StringBuilder(text).reverse().toString(); With the current code, your code will call recursively in a infinite loop, which will finally lead to stack overflow. Please try for you self to see this stack overflow (which is very important to know) and explain why I said this is an infinite loop.